src/core/DY.Core.Bytes.fst

```diff
@@ -226,7 +254,7 @@ let rec get_usage #cusages b =
 
 #push-options "--ifuel 2"
 [@@"opaque_to_smt"]
-val get_label: {|crypto_usages|} -> bytes -> GTot label
+val get_label: {|crypto_usages|} -> bytes -> label
```

Comment:
Why is it not `GTot` anymore?

Reply:
I think this was mentioned at some point during the day at Stuttgart: we would like it to be `GTot`, and it works here on simple examples when it is `GTot`, but we will run into problems for more complex protocols, because the label is not erased in `trace` (constructor `RandGen`) or in `bytes` (constructor `Rand`), and it cannot be easily erased.
This is useful for example in a protocol where you want to do a (symmetric) key exchange: Alice send a public key to Bob, Bob generates a symmetric key and encrypt it to Alice: when generating the symmetric key, Bob doesn't need to know the identity of Alice, and can set the symmetric key label to the join of Bob and `get_sk_label` of the public key, which is possible only when the label computation is not ghost.
This is why `get_label` (or `get_usage`) is not ghost in DY*-intrinsic.
For this PR, I don't think I need to remove `GTot` for labels, but I need to remove it for usages, because the SplitFunctions module produce `Tot` functions and not `GTot` functions

Reply:
Oh I see. Fine. Removing it for both labels and usages seems ok.